ocw-cli/cli_app.py

```diff
@@ -35,7 +36,10 @@
 from ocw.data_source.local import load_file
 import ocw.utils as utils
 import ocw.data_source.esgf as esgf
-from ocw_config_runner.configuration_writer import export_evaluation_to_config
+from example_package import *
+
+#from ocw_config_runner.configuration_writer import export_evaluation_to_config
+from configuration_writer import export_evaluation_to_config
```

Comment:
how come the cli_app.py knows where is configuration_write function without calling "ocw_config_runner.configuration_writer" ?
Yes we need to fix the name of folder from ocw-config-runner to ocw_config_runner, but you should have the folder name (ocw_config_runner) in import